lib/mail/body.rb

```diff
@@ -50,6 +50,11 @@ def initialize(string = '')
       set_charset
     end
 
+    def init_with(coder)
+      coder.map.each { |k, v| instance_variable_set(:"@#{k}", v) }
```

Comment:
Can we call `super` to pull in this implementation, or do we need to implement it ourselves?

Reply:
There's no implementation of it in the super class to call so I left it out.